<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewConnectionDialog</class>
 <widget class="QDialog" name="NewConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Collection Information</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabIdentificationDialog">
      <attribute name="title">
       <string>Identification</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="toolTip">
            <string>Identifier for the collection that is unique across the provider.</string>
           </property>
           <property name="text">
            <string>Id</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="license_la">
           <property name="text">
            <string>License</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="title_le">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>A short descriptive one-line title for the collection.</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="toolTip">
            <string>Detailed multi-line description to fully explain the collection.</string>
           </property>
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="license_le">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="toolTip">
            <string>A short descriptive one-line title for the collection.</string>
           </property>
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="id_le">
           <property name="toolTip">
            <string>Identifier for the collection that is unique across the provider.</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QTextEdit" name="description_le">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Detailed multi-line description to fully explain the collection.</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabKeywordsDialog">
      <attribute name="title">
       <string>Keywords</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>List of keywords describing the collection.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="keywords_table">
         <property name="toolTip">
          <string>A set of descriptive keywords associated with the resource for a specified concept.</string>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Keywords</string>
          </property>
          <property name="toolTip">
           <string>List of keywords describing the Collection</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabExtentDialog">
      <attribute name="title">
       <string>Extent</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Collections spatial and  temporal extent</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsExtentGroupBox" name="spatialExtentSelector">
         <property name="title">
          <string>Spatial extent</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBox" name="mGroupBox">
         <property name="title">
          <string>Temporal extent</string>
         </property>
         <property name="collapsed">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="label_33">
              <property name="text">
               <string>From</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsDateTimeEdit" name="from_date"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLabel" name="label_34">
              <property name="text">
               <string>To</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QgsDateTimeEdit" name="to_date"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_35">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLinksDialog">
      <attribute name="title">
       <string>Links</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="mLabelLinks">
         <property name="toolTip">
          <string>a list of online resources associated with the resource.</string>
         </property>
         <property name="text">
          <string notr="true">List of available links in the collection.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="links_table">
         <property name="toolTip">
          <string>A list of references to other documents.</string>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Title</string>
          </property>
          <property name="toolTip">
           <string>A readable title to be used in rendered displays of the link.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Href</string>
          </property>
          <property name="toolTip">
           <string>Type of address, e.g 'postal'</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
          <property name="toolTip">
           <string>Media type supplied by the link.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Rel</string>
          </property>
          <property name="toolTip">
           <string>he actual link in the format of an URL</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Providers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>List of organizations that captures or processes the content of the collection.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="providers_table">
         <property name="toolTip">
          <string>A list of providers, which may include all organizations capturing or processing the data or the hosting provider.</string>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
          <property name="toolTip">
           <string>The name of the organization or the individual.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
          <property name="toolTip">
           <string>Multi-line description to add further provider information.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Roles</string>
          </property>
          <property name="toolTip">
           <string comment="Roles of the provider. Any of licensor, producer, processor or host.">Postal (or ZIP) code</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>URL</string>
          </property>
          <property name="toolTip">
           <string extracomment="Homepage on which the provider describes the dataset and publishes contact information.">City or locality name</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDateTimeEdit</class>
   <extends>QDateTimeEdit</extends>
   <header>qgsdatetimeedit.h</header>
  </customwidget>
  <customwidget>
   <class>QgsExtentGroupBox</class>
   <extends>QgsCollapsibleGroupBox</extends>
   <header>qgsextentgroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
